runtime/arc.js

```diff
@@ -102,11 +102,12 @@ class Arc {
     return [...this._viewsById.values()];
   }
 
-  clone() {
+  // Makes a copy of the arc used for speculative execution.
+  makeSpeculative() {
```

Comment:
somewhat related 
https://github.com/PolymerLabs/arcs/blob/596fc4ea726e98a1f1e019489aa5f36dfa566349/runtime/arc.js#L77 (i don't know how to comment directly on that line, if it wasn't change in this PR)

var particle = arc.instantiateParticle(serializedParticle.name);

seems to expect instantiateParticle to return the particle, instead it returns a handle to the particle. we should either update the variable name, or the return value - which one is preferable?